lib/id3c/cli/command/etl/fhir.py

```diff
@@ -860,7 +869,8 @@ def process_address(location: Location, tract_hierarchy: str) -> Optional[Any]:
     upsert_encounter_location(db,
         encounter_id    = encounter_id,
         relation        = relation,
-        location_id     = address.id)
+        location_id     = address.id if address else tract_id)
```

Comment:
You could just use `tract.id` directly here instead of creating a new `tract_id` variable.